scenarioo-client/test/e2e/specs/manage_comparison.ts

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import { scenario, step, useCase } from 'scenarioo-js';
+import * as Utils from '../util';
+import ComparisonDetailDialog from '../pages/comparisonDetailLogDialog';
+import ComparisonsPage from '../pages/comparisonsPage';
+
+useCase('Manage comparisons')
+    .description('View and manage comparisons')
+    .describe(() => {
+
+        beforeEach(async () => {
+            await Utils.startScenariooRevisited();
+        });
+
+        scenario('Filter comparisons')
+            .description('Comparisons can be filtered')
+            .it(async () => {
+                await ComparisonsPage.goToPage();
+                const comparisonsCount = ComparisonsPage.getNumberOfComparisons();
+                await step('comparisons page');
+
+                await ComparisonsPage.filterComparisons('To Projectstart');
+                await ComparisonsPage.assertNumberOfComparisons(5);
+                await step('filtered Comparisons for text \'To Projectstart\'');
+
+                await ComparisonsPage.clickResetButton();
+                await ComparisonsPage.assertNumberOfComparisons(await comparisonsCount);
+                await step('reset comparisons page');
+            });
+
+        scenario('Recalculate comparison')
+            .description('Comparisons can be recalculated')
+            .it(async () => {
+                await ComparisonsPage.goToPage();
+                await ComparisonsPage.assertComparisonStatus(0, 'SUCCESS');
+                await step('comparison page with successful comparison');
+
+                await ComparisonsPage.recalculateComparison(0);
+                await ComparisonsPage.assertComparisonStatus(0, 'PROCESSING');
+                await step('comparison in status PROCESSING');
+            });
+
+        scenario('Comparison details and log')
```

Comment:
This test is a change detector.

Reply:
Sorry, the github code snippet doesn't clarify whether scenario "Recalculate comparison" or scenario "Comparison details and log" is meant.
What should be changed?